src/test/java/com/kainos/librarysystem/database/LibraryConnectorTest.java

```diff
@@ -0,0 +1,33 @@
+package com.kainos.librarysystem.database;
+
+import static org.junit.Assert.assertFalse;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.kainos.librarysystem.model.Book;
+
+public class LibraryConnectorTest {
+
+	LibraryConnector lc = new LibraryConnector(
+			"jdbc:mysql://localhost/LibraryDB", "libraryuser",
+			"librarypassword");
+
+	@Test
+	public void testCanGetBooksFromDatabase() {
+
+		List<Book> books = new ArrayList<Book>();
+
+		try {
+			books = new ArrayList<Book>(lc.getBooks());
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+
+		assertFalse(books.size() == 0);
+	}
```

Comment:
Appreciate the attempt at a test but this will fail to deploy on the build server as it has a dependency on the db running. Can you @Ignore it?